gn_build.sh

```diff
@@ -130,6 +136,19 @@ else
     echo "(cd $CHIP_ROOT/examples/lock-app/efr32; gn gen out/debug; ninja -C out/debug)"
 fi
 
+echo
+
+# P6 Build setup
+if [[ -z "$p6_builds_enabled" ]]; then
+    echo "Hint: Pass enable_p6_builds=true to this script to enable building for PSoC6-43012"
+else
+    p6_sdk_args=""
+    if [[ -z "$p6_board_selected" ]]; then
+        p6_sdk_args="p6_board=\"CY8CKIT-062S2-43012\""
+    fi
+    extra_args+=" $p6_sdk_args enable_default_builds=false"
```

Comment:
it was added initially so that p6_build (./gn_build.sh enable_p6_builds=true) will not build all other tools like chip-tool, chip-cert etc.
But removed this now.

Reply:
Make sense, but it should be the user's choice.